documentation-samples/multinic/prerequisites.json

```diff
@@ -0,0 +1,573 @@
+                }
+              ],
+              "loadBalancerInboundNatRules": [
+                {
+                  "id": "[resourceId('Microsoft.Network/loadBalancers/inboundNatRules', parameters('lbName'),concat('RDP-', variables('webVMSetting').vmName, copyIndex(1)))]"
+                }
+              ]
+            }
+          }
+        ]
+      }
+    },
+    {
+      "apiVersion": "2015-06-15",
+      "type": "Microsoft.Compute/availabilitySets",
+      "name": "[variables('webVMSetting').avsetName]",
+      "location": "[variables('location')]",
+      "tags": {
+        "displayName": "AvailabilitySet - Web"
+      }
+    },
+    {
+      "apiVersion": "2015-06-15",
+      "type": "Microsoft.Compute/virtualMachines",
+      "name": "[concat(variables('webVMSetting').vmName,copyindex(1))]",
+      "location": "[variables('location')]",
+      "dependsOn": [
+        "[concat('Microsoft.Network/networkInterfaces/', variables('webVMSetting').nicName, copyindex(1))]",
+        "[concat('Microsoft.Compute/availabilitySets/', variables('webVMSetting').avsetName)]",
+        "[concat('Microsoft.Storage/storageAccounts/', parameters('stdStorageName'))]"
+      ],
+      "tags": {
+        "displayName": "VMs - Web"
+      },
+      "copy": {
+        "name": "webvmcount",
+        "count": "[parameters('webCount')]"
+      },
+      "properties": {
+        "availabilitySet": {
+          "id": "[resourceId('Microsoft.Compute/availabilitySets', variables('webVMSetting').avsetName)]"
+        },
+        "hardwareProfile": {
+          "vmSize": "[variables('webVMSetting').vmSize]"
+        },
+        "osProfile": {
+          "computerName": "[concat(variables('webVMSetting').vmName,copyindex(1))]",
+          "adminUsername": "[variables('webVMSetting').adminUsername]",
+          "adminPassword": "[variables('webVMSetting').adminPassword]"
+        },
+        "storageProfile": {
+          "imageReference": {
+            "publisher": "[variables('webVMSetting').publisher]",
+            "offer": "[variables('webVMSetting').offer]",
+            "sku": "[variables('webVMSetting').sku]",
+            "version": "[variables('webVMSetting').version]"
+          },
+          "osDisk": {
+            "name": "[concat(variables('webVMSetting').osdisk,copyindex(1))]",
+            "vhd": {
+              "uri": "[concat('http://', parameters('stdStorageName'), '.blob.core.windows.net/', variables('vmStorageAccountContainerName'), '/', variables('webVMSetting').osdisk, copyindex(1),'.vhd')]"
+            },
+            "caching": "ReadWrite",
+            "createOption": "FromImage"
+          }
+        },
+        "networkProfile": {
+          "networkInterfaces": [
+            {
+              "id": "[resourceId('Microsoft.Network/networkInterfaces', concat(variables('webVMSetting').nicName,copyindex(1)))]"
+            }
+          ]
+        }
+      }
+    },
+    //{
```

Comment:
I can. However, this works for ARM templates. I will leave it in my local copy.

The reason it is commented is that I cannot get extensions to work, at all. Even using the template you published. DO you have some time to go over your extension template with me later?

From: Kay Singh [mailto:notifications@github.com]
Sent: Thursday, October 29, 2015 10:23 AM
To: Azure/azure-quickstart-templates <azure-quickstart-templates@noreply.github.com>
Cc: Telmo Sampaio <telmos@microsoft.com>
Subject: Re: [azure-quickstart-templates] Added multiNIC documentation sample (#768)


In documentation-samples/multinic/prerequisites.json<https://na01.safelinks.protection.outlook.com/?url=https%3a%2f%2fgithub.com%2fAzure%2fazure-quickstart-templates%2fpull%2f768%23discussion_r43416359&data=01%7c01%7ctelmos%40microsoft.com%7cebdd09b8d2964c9680b708d2e085999a%7c72f988bf86f141af91ab2d7cd011db47%7c1&sdata=2rppy5mVKj%2bz5jUtHpBudnko2v%2fdsufSCHTED4D9R2w%3d>:

> +              "uri": "[concat('http://', parameters('stdStorageName'), '.blob.core.windows.net/', variables('vmStorageAccountContainerName'), '/', variables('webVMSetting').osdisk, copyindex(1),'.vhd')]"

> +            },

> +            "caching": "ReadWrite",

> +            "createOption": "FromImage"

> +          }

> +        },

> +        "networkProfile": {

> +          "networkInterfaces": [

> +            {

> +              "id": "[resourceId('Microsoft.Network/networkInterfaces', concat(variables('webVMSetting').nicName,copyindex(1)))]"

> +            }

> +          ]

> +        }

> +      }

> +    },

> +    //{

JSON doesn't allow commenting out code. Can you remove these?

—
Reply to this email directly or view it on GitHub<https://na01.safelinks.protection.outlook.com/?url=https%3a%2f%2fgithub.com%2fAzure%2fazure-quickstart-templates%2fpull%2f768%2ffiles%23r43416359&data=01%7c01%7ctelmos%40microsoft.com%7cebdd09b8d2964c9680b708d2e085999a%7c72f988bf86f141af91ab2d7cd011db47%7c1&sdata=evun4HGi%2bh4CAtJ8R1I9L7pvYYbqHg1%2fIu6AEfiOwEo%3d>.